e2e/testcases/helm_sync_test.go

```diff
@@ -83,6 +84,39 @@ func TestHelmARFleetWISameProject(t *testing.T) {
 	})
 }
 
+// TestHelmARTokenAuth verifies Config Sync can pull Helm chart from private Artifact Registry with Token auth type.
+// This test will work only with following pre-requisites:
+// Google service account `e2e-test-ar-reader@stolos-dev.iam.gserviceaccount.com` is created with `roles/artifactregistry.reader` for accessing images in Artifact Registry.
```

Comment:
I'd like to start translating these manually provisioned test resources to scripts so that we can easily automate creation of new test environments. None of the other tests have it though so it's not necessary for this PR

Reply:
Sounds good!